service-api/module/Application/src/Controller/Factory/HealthcheckControllerFactory.php

```diff
@@ -31,13 +35,19 @@ public function __invoke(
         $ssmServiceAvailability = getenv("AWS_SSM_SERVICE_AVAILABILITY");
         $config = $container->get('Config');
 
+        $streamHandler = new StreamHandler('php://stderr', LogLevel::INFO);
+        $streamHandler->setFormatter(new OpgFormatter());
+
+        $logger = new Logger('opg-paper-identity/api', [$streamHandler]);
+
         /**
          * @psalm-suppress PossiblyFalseArgument
          */
         return new HealthcheckController(
             $container->get(DataQueryHandler::class),
-            $container->get(SsmClient::class),
+            $container->get(SsmHandler::class),
             $ssmServiceAvailability,
+            $logger,
```

Comment:
There's already a logger setup, so you can fetch it from the container rather than creating a new one:
```suggestion
            $container->get(LoggerInterface::class),
```